### PR TITLE
Add gazebo8 to rosdep

### PR DESCRIPTION
**Pull request by j-rivero**
//cc @mikaelarguedas

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -774,11 +774,19 @@ gazebo:
     trusty: [gazebo2]
     wily: [gazebo7]
     xenial: [gazebo7]
+    zesty: [gazebo8]
```
Comment (mikaelarguedas) [inline rosdep/base.yaml]: you'll need a yakkety entry here for completeness

**Review by mikaelarguedas — APPROVED** [review]
lgtm thanks @j-rivero

**Comment by tfoote** [comment]
Are these already imported and available in the ROS debian repo? I haven't seen tickets for that.

**Comment by mikaelarguedas** [comment]
Good point I forgot to mention it here, thay have not been pulled in yet. thanks for pointing it out.

**Comment by mikaelarguedas** [comment]
requires https://github.com/ros-infrastructure/reprepro-updater/pull/40 before being merged

**Review by tfoote — APPROVED** [review]
I imported into the bootstrap repo, and then into the buildfarm: http://build.ros.org:8080/job/import_upstream/116/console

On Fedora it's gazebo8 by default on F26. So I guess aliasing like this is best.

**Pull request merged**